leap_year.py

```diff
@@ -4,5 +4,17 @@
 
 # Don't forget to reach out for help after your own due diligence
 
-def is_leap_year(year):
-    return ''
\ No newline at end of file
+def is_leap(year):
```

Comment:
The automated tests will fail because they are looking for the function name `is_leap_year`, not `is_leap`